context-aware-reader.md

```diff
@@ -0,0 +1,300 @@
+# Context-aware readers
+
+**[You can find all the code here](https://github.com/quii/learn-go-with-tests/tree/master/q-and-a/cancel-readers)**
```

Comment:
Looks like you couldn't decide to use singular and plural :p
- this link will not work - points to `cancel-readers`, while the directory is `cancel-reader`
- also the production file is called `cancel_reader.go` while the test file is `cancel_readers_test.go`